samples/servicecontrol/fix-messages-using-behavior/sample.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Repair malformed messages using pipeline behavior
+summary: A sample showing how repair malformed messages by implementing a pipeline behavior
+component: Core
+reviewed: 2019-01-07
+---
+
+This sample shows how to repair a malformed message by implementing a pipeline behavior. The sample uses the [Learning Transport](/transports/learning/) and a portable version of the Particular Service Platform tools. Installing ServiceControl is **not** required.
+
+include: platformlauncher-windows-required
+
+downloadbutton
+
+
+## Running the project
+
+Running the project will result in 4 console windows. Wait a moment until the ServicePulse window opens in the browser.
+
+ 1. Press <kbd>Enter</kbd> in the Sender console window.
+ 2. Observe error log output in the Receiver console window: the endpoint cannot process the message because the `Id` field is malformed - it contains lowercase characters.
+ 3. Switch to ServicePulse. It should show one failed message. Hit the retry button and wait until the message is retried.
+ 4. Observe error log output in the Receiver console window. The endpoint still can't process the message.
+ 5. Stop the Receiver project.
+ 5. Update the Receiver configuration code to register the `FixMessageIdBehavior` behavior in the pipeline.
```

Comment:
pardon my OCD: `5` is used two times ;-)

should we mention that the user has to uncomment the code in the `RegisterFixBehavior` region?

Reply:
My bad, sorry. Will fix that. And yes, makes total sense to be explicit about uncommenting. 👍

Reply:
My sample was under SC samples rather than Pipeline samples.
@SzymonPobiega you have no OCD 😃
But you are now tasked to review and if all good, merge.